pom.xml

```diff
@@ -74,10 +74,9 @@
         <junit.version>5.3.2</junit.version>
         <mrunit.version>1.1.0</mrunit.version>
         <mockito.version>2.23.0</mockito.version>
-        <cassandra.version>2.1.20</cassandra.version>
+        <cassandra.version>2.2.13</cassandra.version>
         <jamm.version>0.3.0</jamm.version>
-        <metrics2.version>2.1.2</metrics2.version>
-        <metrics3.version>3.0.1</metrics3.version>
+        <metrics.version>3.2.2</metrics.version>
```

Comment:
Can we use a newer version? Something like `3.2.6` or `4.0.5`?

Reply:
We used this version, because it's already used in `janusgraph-solr` and the DataStax Cassandra driver also used this version (there could be more dependencies that I don't remember right now).
A 4.* version will probably not work as the other dependencies expect a 3.2.* version which probably leads to problems due to breaking changes introduced in the 4.* versions. I could try if it works with 3.2.6, but then I'll have to add more dependency exclusions to avoid the dependency conflicts.

Reply:
@FlorianHockmann if so then it makes sense to use `3.2.2` also.

Reply:
After your question I looked again where exactly the dependency is used in the different modules and found 2 places where it could be simplified: one now unnecessary dependency exclusion and one dependency that isn't needed anymore. I pushed this as a separate commit in case you want to review that so you don't have to review everything again, but I'll squash these two commits before merging.

Reply:
Thanks for the commit. Changes looks good.